static/js/login_or_signup.js

```diff
@@ -3,22 +3,23 @@ var getInputField = function($form, field) {
 };
 
 var getInputFields = function($form, fields) {
-    return fields.map(
-        (field) => getInputField($form, field));
+    return fields.map((field) => getInputField($form, field));
 };
 
 var checkFormValidity = function($form, fields) {
-    return getInputFields($form, fields).every(
```

Comment:
Curious why remove this cascade methodology?

Reply:
It seemed the cleanest, most readable solution to not have this line be over 80 chars long